app/services/fastpass/FastPassService.scala

```diff
@@ -60,6 +69,19 @@ trait FastPassService extends EventSink {
     else {rejectFastPassCandidate(userId, applicationId, actionTriggeredBy)}
   }
 
+  def promoteToFastPassCandidate(applicationId: String, actionTriggeredBy: String)
+                                (implicit hc: HeaderCarrier, rh: RequestHeader): Future[String] = {
+
+    val eventMap = Map("createdBy" -> actionTriggeredBy, "applicationId" -> applicationId)
+    for {
+      _ <- csedRepository.update(applicationId, fastPassDetails)
+      _ <- appRepo.addProgressStatusAndUpdateAppStatus(applicationId, ProgressStatuses.FAST_PASS_ACCEPTED)
+      _ <- eventSink(model.events.AuditEvents.ApplicationReadyForExport(eventMap) :: ApplicationReadyForExport(applicationId) :: Nil)
+      _ <- eventSink(FastPassUserAccepted(eventMap) :: FastPassApproved(applicationId, actionTriggeredBy) :: Nil)
+
+    } yield applicationId
```

Comment:
Do we need to return the app id?  We are just ignoring it in the calling method.